website/src/pages/pricing.tsx

```diff
@@ -32,10 +32,9 @@ export default ((props: any) => (
             </Helmet>
             <div className="pricing-page">
                 <ContentSection color="primary" className="hero-section text-center py-5">
+                    <h1>Sourcegraph pricing</h1>
                     <h3>
-                        Free to start.
-                        <br />
-                        No commitment.
+                        Free to start.  No commitment.
```

Comment:
Looks like there should be a `<br/>` after `Free to start`.

Reply:
I wanted to limit the number of lines in the header i.e. less stacked.